apps/web/src/pages/Profile/ResumeAndRecruitmentsPage/components/ResumeAndRecruitments.tsx

```diff
@@ -0,0 +1,229 @@
+              {intl.formatMessage(titles.manageYourResume)}
+            </Heading>
+            <p>
+              {intl.formatMessage({
+                defaultMessage:
+                  "This section is similar to your traditional résumé and describes your experiences across work, school, and life. You’ll be able to reuse this information on each application you submit on the platform, speeding up the process and ensuring that your information is always up-to-date.",
+                id: "gPy2MA",
+                description:
+                  "Descriptive paragraph for the Manage your resume section of the résumé and recruitments page.",
+              })}
+            </p>
+            {missingSkills && (
+              <div data-h2-margin="base(x1, 0)">
+                <MissingSkills
+                  addedSkills={
+                    hasResumeItems ? flattenExperienceSkills(experiences) : []
+                  }
+                  requiredSkills={missingSkills.requiredSkills}
+                  optionalSkills={missingSkills.optionalSkills}
+                />
+              </div>
+            )}
+            <div data-h2-margin-top="base(x2)">
+              <ResumeSection
+                editParam={applicationParam}
+                experiences={experiences}
+                headingLevel="h2"
+                applicantId={applicantId}
+              />
+            </div>
+          </TableOfContents.Section>
+          <TableOfContents.Section
+            id={PAGE_SECTION_ID.QUALIFIED_RECRUITMENT_PROCESSES}
+            data-h2-margin="base(x3, 0, 0, 0)"
+          >
+            <div data-h2-flex-grid="base(center, x1, x1)">
+              <Heading
+                Icon={IdentificationIcon}
+                color="secondary"
+                data-h2-flex-item="base(1of1) p-tablet(fill)"
+              >
+                {intl.formatMessage(titles.qualifiedRecruitmentProcesses)}
+              </Heading>
+              <Link
+                href={paths.browsePools()}
+                data-h2-flex-item="base(1of1) p-tablet(content)"
+              >
+                Browse jobs
+              </Link>
+            </div>
+            <p data-h2-margin="base(x1, 0, x2, 0)">
+              {intl.formatMessage({
+                defaultMessage:
+                  "When you apply to a recruitment process and successfully pass the assessment, you’re awarded entry and can start being considered for related opportunities. This section highlights all active and expired processes that you’re currently a part of and allows you to manage whether or not you appear in talent searches.",
+                id: "wrTfp3",
+                description:
+                  "Descriptive paragraph for the Qualified recruitment processes section of the résumé and recruitments page.",
+              })}
+            </p>
+            <QualifiedRecruitmentsSection
+              applications={applications}
+              headingLevel="h2"
```

Comment:
I believe we want to use the default `headingLevel` here since this appears as a section within an existing `h2`